src/SFA.DAS.FAA.Web/Controllers/Apply/TrainingCoursesController.cs

```diff
@@ -163,5 +165,47 @@ public async Task<IActionResult> PostEdit(EditTrainingCourseViewModel request)
         await mediator.Send(command);
 
         return RedirectToRoute(RouteNames.ApplyApprenticeship.TrainingCourses, new { request.ApplicationId });
+    }
+
+    [HttpGet]
+    [Route("apply/{applicationId}/trainingcourses/{trainingCourseId}/delete", Name = RouteNames.ApplyApprenticeship.DeleteTrainingCourse)]
+    public async Task<IActionResult> Delete([FromRoute] Guid applicationId, Guid trainingCourseId)
+    {
+        var result = await mediator.Send(new GetDeleteTrainingCourseQuery
+        {
+            ApplicationId = applicationId,
+            CandidateId = Guid.Parse(User.Claims.First(c => c.Type.Equals(CustomClaims.CandidateId)).Value),
```

Comment:
The candidate id can be got via an extension method. 
`User.Claims.CandidateId()`